blog/not-so-basic-base-r-functions/index.qmd

```diff
@@ -93,14 +89,13 @@ vars <- c("col1", "col2")
 
 mtcars %>% tab(cyl)
```

Comment:
Suggest to delete this line: `mtcars %>% tab(cyl)`. It was added when we were debugging tidytab.